bedrock/externalpages/templates/externalpages/pocket/tos.html

```diff
@@ -109,7 +109,7 @@ <h2>Proprietary rights notices</h2>
-      <p><strong class="uppercase">Reasonable efforts are taken to improve the accuracy and integrity of the pocket technologies, but we are not responsible for mistakes, out-of-date information, inaccuracies, typographical or other errors. notwithstanding any other provision of this agreement, we reserve the right to change, suspend, remove, or disable access to any of the pocket technologies, content, or other materials comprising a part of the pocket technologies at any time without notice. in no event will we be liable for making these changes. as a registered user in good standing, you may be provided with limited access to the videos, articles or other content saved to pocket. such access is provided as an accommodation only, and the items you save with pocket may be deleted at any time and for any reason. we do not warrant, and will not have any liability or responsibility for, your use of pocket or the products or services we provide. we may also impose limits on the use of or access to certain features or portions of the pocket technologies, for any reason and without notice or liability. our website, products and services may be unavailable from time to time due to mechanical, telecommunication, software, and other failures. we cannot predict or control when such downtime may occur and cannot control the duration of such downtime.</strong></p>
+      <p><strong class="uppercase">"The pocket technologies (including, without limitation, our application, software, products, services and website) are provided "as is" and with all faults. We make no representation or warranty whatsoever regarding the completeness, accuracy, adequacy, suitability, functionality, availability, or operation of the pocket technologies. you acknowledge that we do not have control over your use of the pocket technologies, and we do not warrant the performance or results that may be obtained through your use of the pocket technologies. you assume all risks and responsibility for your use of the pocket technologies and for any loss of or errors in any data or information. to the full extent permissible by applicable law, we disclaim all warranties, express or implied, including, but not limited to, the implied warranties of merchantability, fitness for a particular purpose, and noninfringement, and any warranties arising through course of dealing or usage of trade. without limiting the foregoing, we neither warrant nor represent that your use of the pocket technologies will not infringe the rights of any third parties, nor that the pocket technologies will be available for your access or use, nor that operation of the pocket technologies will be error free or uninterrupted. please note that some jurisdictions may not allow the exclusion of implied warranties, so some of the above exclusions may not apply to you. in addition, you may also have other rights which vary from state to state."</strong></p>
```

Comment:
Remove start and end quotes for this `p` tag

Reply:
oh thanks! you the best